1-models/articles-model.js

```diff
@@ -0,0 +1,36 @@
+const db = require("../db/connection");
+
+exports.selectArticles = () => {
+  const queryString = `SELECT
+            articles.author,
+            articles.title,
+            articles.article_id,
+            articles.topic,
+            articles.created_at,
+            articles.votes,
+            articles.article_img_url,
+            COUNT(comments.comment_id) AS comment_count
+        FROM articles
+        LEFT JOIN comments ON articles.article_id = comments.article_id
+        GROUP BY articles.author, articles.title, articles.article_id
+        ORDER BY articles.created_at DESC;`;
+
+  return db.query(queryString).then(({ rows }) => {
+    return rows.map((article) => ({
+      ...article,
+      comment_count: parseInt(article.comment_count),
```

Comment:
Strongly advise converting comment count to a number using SQL instead of javascript for the sake of performance and avoiding possibility of mutation - see here: https://www.w3schools.com/sql/func_sqlserver_cast.asp